CMakeLists.txt

```diff
@@ -5,7 +5,18 @@ project(tenseal)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 
-add_subdirectory(third_party/SEAL)
+# For windows we should build SEAL using Microsoft Visual Studio
+# then link the library
+if (WIN32)
+    # including SEAL headers only
+    include_directories(third_party/SEAL/native/src)
+    # linking with the already build seal.lib via MSVC
+    LINK_LIBRARIES(${PROJECT_SOURCE_DIR}/third_party/SEAL/lib/x64/Release/seal.lib)
```

Comment:
Would be better to put this at the bottom, to have inclusions in a block, and linking in another

Reply:
No, we can't, this should be before `pybind11_add_module`.